styles.css

```diff
@@ -746,3 +756,188 @@ ul {
 .border :hover {
   border: 1px solid #396df2;
 }
+
+.contact_me {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 5410px;
```

Comment:
- [OPTIONAL] Please ensure to fix the image background position for the contact form as shown in the Figma Design

### Your Design
![image](https://user-images.githubusercontent.com/68030297/220992177-24851dc7-e734-400a-a202-8bd28c9f246f.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/220992262-ba9e7da6-3861-43e6-8e4a-7d67cd79f2f6.png)